tests/hubapi/demohub/biothing_studio/hub/databuild/mapper.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, *args, **kwargs):
 
     def load(self):
         if self.cache is None:
-            col = mongo.get_src_db()[TaxonomyNodesUploader.name]
+            # This must be the name of TaxonomyNodesUploader that defined on the taxomony plugin manifest file.
+            col = mongo.get_src_db()['nodes']
```

Comment:
hard-coding "nodes" here is fine, maybe add the link to the exact line in manifest defining `name: nodes` to be clear.